drivers/edac/ibecc.h

```diff
@@ -7,9 +7,8 @@
 /* TODO: Add to include/sys/util.h */
 #define BITFIELD(val, h, l)	(((val) & GENMASK(h, l)) >> l)
 
-#define EDAC_IBECC_NAME		"IBECC"
-
 #define PCI_VENDOR_ID_INTEL	0x8086
+
 #define PCI_DEVICE_ID_SKU7	0x452a
 #define PCI_DEVICE_ID_SKU12	0x4518
```

Comment:
I don't think you should be creating a new global PCI_* namespace, especially since this isn't even a PCI header file. These probably belong inside the c-file.